reports-backend/src/main/java/org/jboss/da/listings/api/model/ProductArtifactRelationship.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.da.listings.api.model;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+@ToString
+public class ProductArtifactRelationship {
```

Comment:
It's not entity hibernate takes care of relationship, this is just a helper class so I can get out of DB pair of product and artifact